superset/utils/core.py

```diff
@@ -1787,8 +1787,25 @@ def indexed(
     return idx
 
 
+def strtobool(val: str) -> bool:
+    """Convert a string representation of truth to true (1) or false (0).
+
+    True values are 'y', 'yes', 't', 'true', 'on', and '1'; false values
+    are 'n', 'no', 'f', 'false', 'off', and '0'.  Raises ValueError if
+    'val' is anything else.
+
+    Original implementation from deprecated distutils.
+    """
+    val = val.lower()
+    if val in {"y", "yes", "t", "true", "on", "1"}:
+        return True
+    if val in {"n", "no", "f", "false", "off", "0"}:
+        return False
+    raise ValueError("invalid truth value %r" % (val,))
```

Comment:
Just a thought: To simplify the code, would it be feasible to just place this logic inside `parse_boolean_string`?  Or at least consolidate these somehow and just use that one util function everywhere..

Reply:
Thanks for your feedback! I consolidated `strtobool` and `parse_boolean_string`